llvm/test/compiler-course/guseynov_e_llvmir/test.ll

```diff
@@ -0,0 +1,80 @@
+; RUN: opt -load-pass-plugin %llvmshlibdir/addReplacePass_Guseynov_Emil_FIIT2_LLVM_IR%pluginext\
```

Comment:
Please, also add the test for the case when there is no `@add` function in the module